cpp/src/bitmask/null_mask.cu

```diff
@@ -94,6 +94,22 @@ rmm::device_buffer create_null_mask(size_type size, mask_state state,
   return mask;
 }
 
+// Set pre-allocated null mask to:
+//ALL_VALID, if valid_flag==true, or ALL_NULL, otherwise;
+void set_null_mask(bitmask_type* ptr_bitmask,
+                   size_type size, bool valid_flag,
+                   cudaStream_t stream)
+{
+  if (ptr_bitmask != nullptr)
+    {
```

Comment:
This indentation looks a little strange.
Also see Mark's suggestions in null_mask.hpp re `ptr_bitmask` and `valid_flag`

Reply:
Done.